packages/inferno/src/global.d.ts

```diff
@@ -0,0 +1,177 @@
+/*
+React projects that don't include the DOM library need these interfaces to compile.
+React Native applications use React, but there is no DOM available. The JavaScript runtime
+is ES6/ES2015 only. These definitions allow such projects to compile with only `--lib ES6`.
+*/
```

Comment:
Why do we need this for inferno?

Reply:
If I remove the reference to global, there are errors with HTMLWebViewElement. Also this would be needed for projects that use Inferno on the server side.